.github/workflows/linters.yml

```diff
@@ -0,0 +1,70 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] You didn't add a LICENSE file for your project :sob: but no problem you can add a license file using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :blush: Kindly add a license file because you have a link to your LICENSE file in your readme file :thinking: